CONTRIBUTING.md

```diff
@@ -196,3 +196,39 @@ In this example this can be done by invoking following command from the reposito
 ```bash
 ./gradlew test -Pcoordinates=org.example:library:0.0.1
 ```
+
+## Tested Libraries and Frameworks
+
+If your library or framework is tested with GraalVM Native Image, consider adding it to [this list](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list.json).
+
+Write an entry as follows:
+```json
+{
+    "artifact": "<groupId>:<artifactId>",
+    "description": "<artifactDescription>",
+    "details": [
+      {
+        "minimum_version": "<minimumVersion>",
+        "maximum_version": "<maximumVersion>",
+        "metadata_locations": ["<metadataLocations>"],
+        "tests_locations": ["<testLocations>"],
+        "test_level": "<testLevel>"
+      }
+    ]
+}
+```
+Where:
+ * `<groupId>` and `<artifactId>` - standard Maven coordinates ([see this](https://maven.apache.org/pom.html#maven-coordinates))
+ * `<artifactDescription>` - short description of the library or framework (_not required_) 
```

Comment:
On second though, I don't think we should document this in this detail more than once. Let's refer to the JSON schema, which already has descriptions such as [this one](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list-schema.json#L54).

Maybe this is enough:

```
## Tested Libraries and Frameworks

If you would like to add your project to the list of libraries or frameworks tested with GraalVM Native Image, please open a pull request and add an entry to [this file](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list.json) according to [this schema](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list-schema.json). Entries are listed on the [GraalVM website](https://graalvm.org/native-image/libraries-and-frameworks/) and may be used for tools in the future.

**Note:** To pass format and style checks, please run `./gradlew :spotlessApply` from the project root, before submitting a PR.  
**Note:** The entries you add will be validated against [library-and-framework-list-schema.json](https://github.com/oracle/graalvm-reachability-metadata/blob/master/library-and-framework-list-schema.json)

Reply:
I disagree here. I would like to get the description in a human-readable form right where my copy-paste example is.

Reply:
Users can copy the example from the JSON and will likely copy existing entries in most cases anyway. I'm against duplicating content but we can keep this one.